app/javascript/hyacinth_ui_v1/components/dynamic_fields/DynamicFieldIndex.js

```diff
@@ -5,73 +5,92 @@ import produce from 'immer';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import ContextualNavbar from '../layout/ContextualNavbar';
-import hyacinthApi from '../../util/hyacinth_api';
+
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+
+import { Can } from '../../util/ability_context';
+import GraphQLErrors from '../ui/GraphQLErrors';
+
+
 import DynamicFieldsAndGroupsTable from '../layout/dynamic_fields/DynamicFieldsAndGroupsTable';
 import EditButton from '../ui/buttons/EditButton';
 
-export default class DynamicFieldIndex extends React.Component {
-  state = {
-    dynamicFieldGraphs: [],
-  }
 
-  componentDidMount() {
-    this.getDynamicFieldCategories();
-  }
+import { Table } from 'react-bootstrap';
+
+
+
+  const DYNAMICFIELDS = gql`
```

Comment:
All caps for a constant is common in other languages, but in JavaScript this is actually an uncommon thing to do for a scoped (in this case, scoped to the file its defined in) variable that just happens to be an unchangeable `const`.  I'd recommend going for regular camelCase here.